app/e2e/english_lookup_forms.spec.js

```diff
@@ -0,0 +1,82 @@
+// https://playwright.dev/docs/writing-tests#first-test
+// @ts-check
+import { test, expect } from '@playwright/test'
+
+const test_data = [
+	{
+		test_word: 'following',
+		expected_forms: [
+			{ stem: 'following', part_of_speech: 'Adjective', form: 'Stem' },
```

Comment:
I suppose these will all need the ids now?

Reply:
I decided against that since the id's can change... that was why I went with that new `toMatchObject` assertion.

Reply:
Ah, I didn't notice `toMatchObject`. Great!